src/Evaluators/reduced_evaluator.jl

```diff
@@ -512,8 +512,8 @@ function hessprod!(nlp::ReducedSpaceEvaluator, hessvec, u, w)
     _second_order_adjoint_z!(nlp, z, w)
 
     # Init tangent
-    tgt[1:nx] .= z
-    tgt[1+nx:nx+nu] .= w
+    copyto!(tgt, 1, z, 1, nx)    # tgt[1:nx] .= z
```

Comment:
To me, it does not have to be explained in a comment anymore.

Reply:
I will remove the comment then

Reply:
Comments have been removed.